Tests/Components Tests/Apple Pay/ApplePayComponentTests.swift

```diff
@@ -269,13 +269,24 @@ class ApplePayComponentTest: XCTestCase {
         let paymentRequest = configuration.createPaymentRequest(supportedNetworks: paymentMethod.supportedNetworks)
         XCTAssertEqual(paymentRequest.paymentSummaryItems, expectedSummaryItems)
         XCTAssertEqual(paymentRequest.merchantCapabilities, PKMerchantCapability.capability3DS)
-        XCTAssertEqual(paymentRequest.supportedNetworks, self.supportedNetworks)
         XCTAssertEqual(paymentRequest.currencyCode, currencyCode)
         XCTAssertEqual(paymentRequest.merchantIdentifier, "test_id")
         XCTAssertEqual(paymentRequest.countryCode, countryCode)
         XCTAssertEqual(paymentRequest.requiredBillingContactFields, expectedRequiredBillingFields)
         XCTAssertEqual(paymentRequest.requiredShippingContactFields, expectedRequiredShippingFields)
     }
+    
+    
+    func testNetworks() {
+        if #available(iOS 15.1, *) {
+            let request = PKPaymentRequest()
+            let collection: [PKPaymentNetwork] = [.discover, .dankort]
+            XCTAssertEqual(collection.count, 2)
+            
+            request.supportedNetworks = collection
+            XCTAssertEqual(request.supportedNetworks.count, 1) // 🤷
```

Comment:
I don't this comment is helping anybody :-D, except it in making the reader smile  :-D

maybe we should add something to explain

Reply:
Sure :)

Reply:
Fixed also [reported here](https://developer.apple.com/forums/thread/715110)